drivers/sensor/bosch/bme280/bme280.c

```diff
@@ -37,6 +37,9 @@ LOG_MODULE_REGISTER(BME280, CONFIG_SENSOR_LOG_LEVEL);
 #define BME280_EXPECTED_SAMPLE_TIME_MS                                                             \
 	1 + BME280_TEMP_SAMPLE_TIME + BME280_PRESS_SAMPLE_TIME + BME280_HUMIDITY_SAMPLE_TIME
 
+/* Number of trials for chip id read */
+#define BME280_CHIP_ID_READ_RETRY_CNT 10
```

Comment:
Do we really need to read a sensor register 10 times to reliably initialize it? It sounds like something else is going on.

Why instead of this don't we just add a sleep() call with the datasheet's specified power-on-time?

Reply:
Ok. Does that fix the issue you were facing?